src/redux/reducers/audioplayer.ts

```diff
@@ -52,7 +52,9 @@ export default (state = INITIAL_STATE, action: $TsFixMe) => {
     case FETCH_AUDIOPLAYER: {
       return handle(state, action, {
         success: prevState => {
-          const audioFile: $TsFixMe = camelcaseKeys(action.payload.audio_file);
+          const audioFile: $TsFixMe = camelcaseKeys(
+            action.payload.audio_file || {}
```

Comment:
this is awesome! perhaps we should use `lodash/get`? so `get(action, 'payload.audio_file', {})` such that if payload is `null`, this would still work